docs/index.md

```diff
@@ -33,195 +33,13 @@ With Prefect you gain:
-    response = httpx.get(contributors_url)
-    response.raise_for_status()
-    contributors = response.json()
-    return contributors
-
-@flow(name="Repo Info", log_prints=True)  
-def repo_info(
-    repo_owner: str = "PrefectHQ", repo_name: str = "prefect"
-):
-    # call our `get_repo_info` task
-    repo_info = get_repo_info(repo_owner, repo_name)
-    print(f"Stars 🌠 : {repo_info['stargazers_count']}")
-
-    # call our `get_contributors` task, 
-    # passing in the upstream result
-    contributors = get_contributors(repo_info)
-    print(
-        f"Number of contributors 👷: {len(contributors)}"
-    )
-
-
-if __name__ == "__main__":
-    # Call a flow function for a local flow run!
-    repo_info()
-```
-
-### Step 4: Run your flow locally
-Call any function that you've decorated with a `@flow` decorator to see a local instance of a flow run.
-
-<div class="terminal">
-```bash
-python my_flow.py
-```
-</div> 
-
-<div class="terminal">
-```bash
-18:21:40.235 | INFO    | prefect.engine - Created flow run 'fine-gorilla' for flow 'Repo Info'
-18:21:40.237 | INFO    | Flow run 'fine-gorilla' - View at https://app.prefect.cloud/account/0ff44498-d380-4d7b-bd68-9b52da03823f/workspace/c859e5b6-1539-4c77-81e0-444c2ddcaafe/flow-runs/flow-run/c5a85193-69ea-4577-aa0e-e11adbf1f659
-18:21:40.837 | INFO    | Flow run 'fine-gorilla' - Created task run 'get_repo_info-0' for task 'get_repo_info'
-18:21:40.838 | INFO    | Flow run 'fine-gorilla' - Executing 'get_repo_info-0' immediately...
-18:21:41.468 | INFO    | Task run 'get_repo_info-0' - Finished in state Completed()
-18:21:41.477 | INFO    | Flow run 'fine-gorilla' - Stars 🌠 : 12340
-18:21:41.606 | INFO    | Flow run 'fine-gorilla' - Created task run 'get_contributors-0' for task 'get_contributors'
-18:21:41.607 | INFO    | Flow run 'fine-gorilla' - Executing 'get_contributors-0' immediately...
-18:21:42.225 | INFO    | Task run 'get_contributors-0' - Finished in state Completed()
-18:21:42.232 | INFO    | Flow run 'fine-gorilla' - Number of contributors 👷: 30
-18:21:42.383 | INFO    | Flow run 'fine-gorilla' - Finished in state Completed('All states completed.')
-```
-</div>
-
-
-You'll find a link directing you to the flow run page conveniently positioned at the top of your flow logs.
-
-![Alt text](img/ui/flow-run-diagram.jpg)
-
-Local flow run execution is great for development and testing, but to [schedule flow runs](/concepts/schedules/) or [trigger them based on events](/cloud/automations/), you’ll need to [deploy](/concepts/deployments/) your flows.
+Get up and running quickly with the [quickstart guide](getting-started/quickstart) and [tutorial](/tutorial/); for deeper dives on specific use cases, explore our [guides](guides/index/) for common scenarios.  Take your understanding even further with Prefect's [concepts](/concepts/index/) and [API reference](/api-ref/).  Join Prefect's [vibrant community](/community/) of over 26,000 engineers to learn with others and share your knowledge!
```

Comment:
```suggestion
Get up and running quickly with the [quickstart guide](/getting-started/quickstart/). 
Want more hands on practice to productionize your workflows? Follow our [tutorial](/tutorial/). 
For deeper dives on common use cases, explore our [guides](/guides/index/). 
Take your understanding even further with Prefect's [concepts](/concepts/index/) and [API reference](/api-ref/). 
Join Prefect's [vibrant community](/community/) of over 26,000 engineers to learn with others and share your knowledge!
```

Reply:
Formatted links for consistency. 

Would like to try to differentiate when to use the quick start vs. tutorial, so added the language above.